src/ipsis/course-merge/CourseMergeOfferingEntity.js

```diff
@@ -40,11 +40,20 @@ export class CourseMergeOfferingEntity extends Entity {
 		return this._entity?.properties?.sectionMappingCount;
 	}
 
+	sectionDeleted() {
+		return this._entity?.properties?.sectionDeleted;
+	}
+
 	onOrganizationChange(onChange) {
 		const organizationHref = this.organizationHref();
+		if (!organizationHref) {
+			// setTimeout to allow caller to finish other work before calling onChange.
+			setTimeout(() => onChange(null), 0);
```

Comment:
I'm calling the callback here with `null` so we don't have to do a bunch of extra checks for whether the `organizationHref` is there. We'll just get the `null` in the callback and set `this._organization` to `null`.